src/style/blocks/card.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  font-family: Roboto, sans-serif;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  width: 200px;
```

Comment:
as per today's lesson, width/margins, etc should be a characteristic of the outer container (according to BEM)